importer/DkBygningDa.py

```diff
@@ -11,20 +11,22 @@ def set_adm_location(self):
                 self.add_statement("located_adm", adm_location)
 
     def set_location(self):
-        """
-        Use self.municipality because IT'S PLACE NOT MUNICIPALITY
-        (that's adm2)
-        """
         place_item = False
         if self.has_non_empty_attribute("by"):
             place = self.by
             if utils.count_wikilinks(place) == 1:
-                place_item = utils.q_from_first_wikilink("da", place)
+                linked_page_title = utils.get_wikilinks(place)[0].title
+                if utils.wp_page_exists("da", linked_page_title):
+                    place_item = utils.q_from_first_wikilink("da", place)
             else:
                 if utils.wp_page_exists("da", place):
                     place_item = utils.q_from_wikipedia("da", place)
         if place_item:
-            self.add_statement("location", place_item)
+            place_item_ids = utils.get_P31(place_item)
+            for is_instance in place_item_ids:
```

Comment:
I would rename `is_instance` (e.g `instance_of` or `p31_value`) to avoid potential confusion with the python method `isinstance`.